dmcontent/content_loader.py

```diff
@@ -231,6 +223,7 @@ def get_field_names(self):
         ]
 
     def get_question_ids(self, type=None):
+        # FIXME should this ever return self.id?
```

Comment:
It's covered by a test of current behaviour, but it's not clear what the expectation is, so I've just marked this for the future.